packages/swap/src/routes/swap.ts

```diff
@@ -221,6 +222,8 @@ router.get(
       if (swap.refundEgress.broadcast?.abortedAt) {
         state = State.Failed;
         failureMode = Failure.RefundBroadcastAborted;
+      } else if (swap.refundEgress.broadcast?.succeededAt) {
+        state = State.Refunded;
```

Comment:
do we have a refund tx hash?

Reply:
should be able to use the one from the broadcast

Reply:
yeah it's a normal broadcast procedure, so we should have